src/us/kbase/workspace/kbase/ArgUtils.java

```diff
@@ -178,40 +176,27 @@ public static Provenance processProvenance(final WorkspaceUser user,
 					.withDataUrl(edu.getDataUrl())
 					.withDescription(edu.getDescription())
 					.withResourceName(edu.getResourceName())
-					.withResourceReleaseDate(d)
+					.withResourceReleaseDate(d == null ? null : Date.from(d))
 					.withResourceUrl(edu.getResourceUrl())
 					.withResourceVersion(edu.getResourceVersion())
 			);
 		}
 		return ret;
 	}
 
-	//TODO CODE why does this throw ParseException? 
-	private static Date parseDate(final String date) throws ParseException {
-		if (date == null) {
-			return null;
-		}
-		try {
-			return DATE_PARSER.parseDateTime(date).toDate();
-		} catch (IllegalArgumentException iae) {
-			throw new IllegalArgumentException("Unparseable date: " +
-					iae.getMessage());
-		}
-	}
-	
+	// TODO CODE remove this eventually when everything uses Instants
 	private static String formatDate(final Date date) {
 		if (date == null) {
 			return null;
 		}
-		return DATE_FORMATTER.print(new DateTime(date));
+		return formatDate(date.toInstant());
```

Comment:
ok -- so it's basically just swapping over to formatting Instant objects rather than Dates. 👍